sycl/source/detail/queue_impl.hpp

```diff
@@ -783,7 +783,6 @@ class queue_impl {
       EventRet = Handler.finalize();
   }
 
-protected:
```

Comment:
Why is this removed?

Reply:
it is a duplication. protected title is already present on l748

Reply:
Thanks for clarifying.